src/screens/GovernanceCall/GovernanceCallBanner.js

```diff
@@ -0,0 +1,222 @@
+
+// local consts
+const TYPE_GOVERNANCE_CALL = 'governance-calls';
+const DATE_FORMAT = 'do LLL'; // Human readable
+const TIME_WITH_TIMEZONE_FORMAT = 'HH:mm O'; // Human readable
+
+const GovernanceCallBanner = () => {
+  const { t } = useTranslationWithPrefix('governanceCall');
+  const [title, setTitle] = React.useState(t('banner.title'));
+  const [description, setDescription] = React.useState('');
+  const [governanceCallLink, setGovernanceCallLink] = React.useState('');
+  const [timingData, setTimingData] = React.useState({});
+  const [remainingTime, setRemainingTime] = React.useState({});
+  const [isVisible, setIsVisible] = React.useState(false);
+
+  const calculateLeftOverTime = (startTime) => {
+    const thisDateNow = Date.now();
+    const daysTillEventStart = differenceInDays(startTime, thisDateNow);
+    const hoursTillEventStart = differenceInHours(startTime, thisDateNow);
+    const minutesTillEventStart = differenceInMinutes(startTime, thisDateNow);
+    return { daysTillEventStart, hoursTillEventStart, minutesTillEventStart };
+  };
+
+  const fetchGovernanceCallDataMemoized = React.useCallback(() => {
+    Prismic.queryDocumentsByType(TYPE_GOVERNANCE_CALL)
+      .then((result) => {
+        const thisDateNow = Date.now();
+        const prismicResponse = result?.results[0]?.data;
+        setTitle(prismicResponse?.title[0]?.text);
+        setDescription(prismicResponse?.description[0]?.text);
+        setGovernanceCallLink(prismicResponse?.link?.url);
+        const governanceCallStartTime = new Date(parseISO(prismicResponse?.scheduled_for));
+        const governanceCallEndTime = addHours(governanceCallStartTime, 1);
+        const timeTillEventStart = calculateLeftOverTime(governanceCallStartTime);
+        setRemainingTime(timeTillEventStart);
+        const isEventLive =
+          !isBefore(governanceCallEndTime, thisDateNow) && timeTillEventStart.minutesTillEventStart <= 0;
+
+        const formattedDate = formatDate(governanceCallStartTime, DATE_FORMAT);
+
+        const formattedTimeWithTimezone = formatDate(governanceCallStartTime, TIME_WITH_TIMEZONE_FORMAT);
+
+        const calculatedBannerVisibility =
+          (prismicResponse?.show_permanently && isAfter(governanceCallEndTime, thisDateNow)) ||
+          (timeTillEventStart.hoursTillEventStart <= 24 && timeTillEventStart.hoursTillEventStart >= 0);
+
+        setTimingData({
+          governanceCallStartTime,
+          governanceCallEndTime,
+          formattedDate,
+          formattedTimeWithTimezone,
+          isEventLive,
+        });
+
+        setIsVisible(calculatedBannerVisibility);
+
+        return true;
+      })
+      .catch((error) => reportErrorLog('Prismic content fetch failed', { error }));
+  }, []);
+
+  React.useEffect(() => {
+    fetchGovernanceCallDataMemoized();
+  }, [fetchGovernanceCallDataMemoized]);
+
+  React.useEffect(() => {
+    setInterval(() => {
+      if (timingData?.governanceCallStartTime !== undefined) {
```

Comment:
Why not just? Since it can be parsed date or nullish
```suggestion
      if (!!timingData?.governanceCallStartTime) {
```

Reply:
Done